src/AMSlib/ml/uq.hpp

```diff
@@ -33,12 +31,17 @@ class UQ
      FPTypeValue threshold)
       : uqPolicy(uqPolicy), threshold(threshold)
   {
+    if (isNullOrEmpty(surrogatePath)) {
```

Comment:
Do we allow the absence of a surrogate? What happens in this case?

Reply:
It fall backs to the original existing physics code and performs data collection.